app/src/androidTest/java/com/example/sharingang/MapActivityTest.kt

```diff
@@ -49,4 +49,4 @@ class MapActivityTest {
         Thread.sleep(10000)
         text.check(matches(withText(containsString("Your location"))))
     }
-}
\ No newline at end of file
+}
```

Comment:
Very useful x)

Reply:
It was added when I reformatted, I think this is good to have new lines at the end of files
https://thoughtbot.com/blog/no-newline-at-end-of-file